docs/console/docs/06-02-clustermicrofrontend.md

```diff
@@ -3,7 +3,7 @@ title: ClusterMicroFrontend
 type: Custom Resource
 ---
 
-The `clustermicrofrontends.ui.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format used to extend the Kyma Console. It allows you to extend the Console for the entire Cluster. To get the up-to-date CRD and show the output in the `yaml` format, run this command:
+The `clustermicrofrontends.ui.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format used to extend the Kyma Console. It allows you to extend the Console for the entire Cluster. ClusterMicroFrontends get added to console automatically based on the yaml. In order to avoid naming colissions with the core system, the root node gets prefixed with `cmf-` in the URL. Additionally `navigationContext` and `viewGroup` [Node configuration](https://github.com/kyma-project/luigi/blob/master/docs/navigation-parameters-reference.md#node-parameters) options are set, to allow simple navigation.To get the up-to-date CRD and show the output in the `yaml` format, run this command:
```

Comment:
```suggestion
The `clustermicrofrontends.ui.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format used to extend the Kyma Console. It allows you to extend the Console for the entire Cluster. The ClusterMicroFrontend CR is added to Console automatically based on the `yaml` file. To avoid naming conflicts with the core system, the root node receives the `cmf-` prefix in the URL. Additionally,  **navigationContext** and **viewGroup**  [node configuration](https://github.com/kyma-project/luigi/blob/master/docs/navigation-parameters-reference.md#node-parameters) parameters are set to allow simple navigation. To get the up-to-date CRD and show the output in the `yaml` format, run this command:
```

Reply:
```
A cluster micro frontend is added to console automatically based on...
```

Reply:
right :)